src/super_gradients/recipes/dataset_params/imagenet_vit_base_dataset_params.yaml

```diff
@@ -0,0 +1,44 @@
+defaults:
+  - imagenet_dataset_params
+
+resize_size: 249
+batch_size: 64
+random_erase_prob: 0
+random_erase_value: random
+train_interpolation: random
+rand_augment_config_string: rand-m7-mstd0.5
+cutmix: True
+cutmix_params:
+  mixup_alpha: 0.2
+  cutmix_alpha: 1.0
+  label_smoothing: 0.1
+img_mean: [0.5, 0.5, 0.5]
+img_std: [0.5, 0.5, 0.5]
+
+
+# TODO: REMOVE ABOVE, HERE FOR COMPATIBILITY UNTIL WE REMOVE DATASET_INTERFACE
+
+# RuntimeError: Default process group has not been initialized, please make sure to call init_process_group.
```

Comment:
What is this ?

Reply:
This was a note while debugging, I removed it